metatron-app/metatron-discovery
# #3774 add org_code column to tenant entity

### Description
Injecting org_code into Tenant Managed Entity

**Related Issue** : #3774 

### How Has This Been Tested?
1. Check whether org_code column is created in Datasource, DataConnection, RoleDirectory, RoleSet, and ActivityStream tables
2. Check if data is updated in user_org_member table when user or group is created or deleted
3. When creating Datasource, DataConnection, RoleDirectory, RoleSet Entity, check if the org_code value of the user is included in org_code

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
